examples/homography_estimation.py

```diff
@@ -0,0 +1,152 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+
+import hydra
+from hydra.utils import get_original_cwd
+import kornia
+import torch.nn as nn
+
+# import theseus as th
+import torch
+from PIL import Image
+import cv2
+import numpy as np
+import os
+
+
+from libs.easyaug import RandomGeoAug, GeoAugParam
```

Comment:
examples/libs/ is okay for now. Once the example is setup with Theseus we can figure out if there are common things worth sharing between other problems. In that case experimental stuff goes in theseus/utils/examples/ and more well tested stuff goes in theseus/embodied. For external code we can probably setup a 3rdparty dir for more widely useful things. For installing extra dependencies you can just indicate that in the examples/README and comments in this file.